mw-webapp/src/logic/waysTable/columns.tsx

```diff
@@ -26,7 +26,13 @@ export const columns = [
     /**
      * Cell with clickable way name that leads to way page
      */
-    cell: ({row}) => renderLinkInCell(`/way/${row.original.uuid}`, row.original.name),
+    cell: ({row}) => (
+      <Link
+        path={`/way/${row.original.uuid}`}
```

Comment:
It's very hard to maintain template strings and it will be easy to make a bug with them. We have a file with pathes, why not to put function which returns some path there? It could be reusable in this way